Core/src/Surfaces/Surface.cpp

```diff
@@ -156,7 +161,11 @@ Acts::Surface& Acts::Surface::operator=(const Surface& other) {
   if (&other != this) {
     GeometryObject::operator=(other);
     // detector element, identifier & layer association are unique
-    m_transform = other.m_transform;
+    if (other.m_transform) {
+      m_transform = std::make_unique<Transform3>(*other.m_transform);
+    } else {
+      m_transform.reset();
```

Comment:
Coverage is now resolved. Just this single untested branch is added. Maybe you can think of a test for it. Otherwise only the formatting fails.

Reply:
If the transform is not there then it's connected with the detector element. Can't think about a test right now which would be sensible